.github/workflows/create_release.yml

```diff
@@ -0,0 +1,63 @@
+name: Create Release
+
+on:
+  push:
```

Comment:
Do we need to specify only the master branch here?

Reply:
Good catch! I actually just had it for any tag that started with v but have made it so it should only be master branch.